src/features/services/queries.ts

```diff
@@ -85,11 +79,11 @@ export function useCreateToken() {
           [inputs.address]: amount,
         },
       };
-      network?.tokens.create(param);
+      return await network?.tokens.create(param);
     },
     onSuccess: () =>
       queryClient.invalidateQueries({
-        queryKey: ["ledger.tokens", network?.url],
+        queryKey: ["tokens"],
```

Comment:
Now we're invalidating every query that "starts" with `["tokens"]`, which will be both the list of all tokens and the information for each one. This will trigger a refresh. This is bug 2.